### DISCUSSION TITLE
How does memtest86+ compare with the Windows Memory Diagnostic Tool?

### DISCUSSION THREAD

**Discussion by ksze** [memtest86plus/memtest86plus#37] [Q&A]
Suppose we want to compare between the new memtest86+ v6 beta against WMDT.

What do users need to know, in terms of test coverage, caveats, etc?

**Comment by BenMyers43** [comment]
WMDT is superficial.  Microsoft is very weak in the area of hardware diagnostics.  I never think of using WMDT or the quick memory tests some system manufacturers build into the motherboard BIOS.  I want comprehensive testing.
